File: USACO/promotionCounting.java
import java.io.*;
import java.util.StringTokenizer;

public class promotionCounting {
    public static void main(String[] args) throws IOException
    {
        BufferedReader r = new BufferedReader(new FileReader("promote.in"));
        PrintWriter pw = new PrintWriter("promote.out");

        StringTokenizer st = new StringTokenizer(r.readLine());
        //int bb = Integer.parseInt(st.nextToken());
		//int ba = Integer.parseInt(st.nextToken());

		st = new StringTokenizer(r.readLine());
		int sb = Integer.parseInt(st.nextToken());
		int sa = Integer.parseInt(st.nextToken());

		st = new StringTokenizer(r.readLine());
		int gb = Integer.parseInt(st.nextToken());
		int ga = Integer.parseInt(st.nextToken());

		st = new StringTokenizer(r.readLine());
		int pb = Integer.parseInt(st.nextToken());
		int pa = Integer.parseInt(st.nextToken());

        int gold = pa - pb;
        int silver = ga - gb + pa - pb;
        int bronze = sa - sb + ga - gb + pa - pb;

        pw.println(bronze);
        pw.println(silver);
        pw.println(gold);
        pw.close();
        r.close();
    }
}

File: USACO/promotionCounting.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    cin.tie(0)->sync_with_stdio(0);
    freopen("promote.in", "r", stdin);
    freopen("promote.out", "w", stdout);

    int sb, sa, gb, ga, pb, pa, bb, ba;
    cin >> bb >> ba >> sb >> sa >> gb >> ga >> pb >> pa;

    int gold = pa - pb;
    int silver = ga - gb + pa - pb;
    int bronze = sa - sb + ga - gb + pa - pb;

    cout << bronze << "\n";
    cout << silver << "\n";
    cout << gold << "\n";
}